### PR TITLE
Updated design for announcement/discussion

### PR DESCRIPTION
**Pull request by sarinali**
### Description
Made design for hi fi in new ticket for announcement

### Reason for Change
CFD-91

### Type of change

- [x] New feature (non-breaking change which adds functionality)

IMG_6003

### CODE REVIEW

**Comment by BakerWJ** [comment]
u just deleted this file in ur last pr btw

**Comment by BakerWJ** [comment]
this file will be deleted by ur last pr when it gets merged in, so gotta rebase and fix once it merges

File: apps/expo/src/components/Discussion.tsx
Comment (BakerWJ) [inline apps/expo/src/components/Discussion.tsx:1]: why are u modifying discussion.tsx? announcement should be a different component

Comment (sarinali) [reply apps/expo/src/components/Discussion.tsx:1]: i brought it back :D

**Review by BakerWJ — DISMISSED** [review]

**Review by BakerWJ — APPROVED** [review]

**Pull request merged**